locales/en.js

```diff
@@ -125,12 +125,18 @@ export default {
   footerAboutURL: 'https://www.canada.ca/en/government/about.html',
   footerAbout: 'About Canada.ca',
 
+  footerTermsAndConditionURLtest:
```

Comment:
I'm not a fan of this naming. Could we instead call it something along the lines of `footerTermsAndConditionsECAS`? Same with the privacy URL.